apps/docs/src/guide/wallet-manager/index.md

```diff
@@ -0,0 +1,17 @@
+# Wallet Manager
+
+The Wallet Manager is a robust tool designed for managing vaults of wallets. It offers robust management of vaults, including support for custom storage and powerful encryption of all held vaults.
```

Comment:
```suggestion
The WalletManager is a robust tool designed for managing vaults of wallets. It offers robust management of vaults, including support for custom storage and powerful encryption of all held vaults.
```

Reply:
@arboleya also needs ticks for `WalletManager`